### ISSUE TITLE
Support changing voice prompts

### ISSUE THREAD

**Issue by vitorio** [SamyPesse/betty#14]
Right now, it looks like all the spoken prompts are specified in JS.  It would be nice if a new profile didn't have to be hardcoded into `profiles/index.js`, and if the prompts could either be text or a recorded message (local or remote URL).

**Comment by SamyPesse** [comment]
Yes a command line option like `betty --profile=./myprofile.json` will be great.
